.gitignore

```diff
@@ -1,2 +1,6 @@
 demo-dist
 node_modules
+*.log
```

Comment:
Definitely not a big deal, but curious about what these `.gitignore` changes are for?